diff --git a/src/frontend/main.ml b/src/frontend/main.ml
index 33e75d5..b6d53a4 100644
--- a/src/frontend/main.ml
+++ b/src/frontend/main.ml
@@ -729,10 +729,23 @@ let arg_spec_list curdir =
 
 
 let setup_root_dirs () =
-  let ds =
-    (match Sys.getenv_opt "HOME" with
-    | None    -> []
-    | Some(s) -> [Filename.concat s ".satysfi"]) @ ["/usr/local/share/satysfi"; "/usr/share/satysfi"] in
+  let runtime_dirs =
+    if Sys.os_type = "Win32" then
+      []
+    else
+      ["/usr/local/share/satysfi"; "/usr/share/satysfi"]
+  in
+  let user_dirs =
+    if Sys.os_type = "Win32" then
+      match Sys.getenv_opt "userprofile" with
+      | None    -> []
+      | Some(s) -> [Filename.concat s ".satysfi"]
+    else
+      match Sys.getenv_opt "HOME" with
+      | None    -> []
+      | Some(s) -> [Filename.concat s ".satysfi"]
+  in
+  let ds = user_dirs @ runtime_dirs in
   (if List.length ds = 0 then
     raise NoLibraryRootDesignation);
   Config.initialize ds
