diff --git a/src/config/discover.ml b/src/config/discover.ml
index ef26121..b74d7bb 100644
--- a/src/config/discover.ml
+++ b/src/config/discover.ml
@@ -41,8 +41,8 @@ let () =
 
     let ocaml_vars =
       List.map
-        (C.C_define.import c ~includes:["caml/config.h"]
-           [ "ARCH_SIXTYFOUR" , Switch ])
+        (* On opam-cross-windows, host word size == target word size. *)
+        [ "ARCH_SIXTYFOUR" , C.C_define.Value.Switch (Int.num_bits = 63) ]
         ~f:(fun (name, v) -> ("JSC_" ^ name, v))
     in
 
